### ISSUE TITLE
Public school isn't marked as "SY".

### ISSUE THREAD

**Issue by davehodg** [opendataedu/open-data-education-datasets#2]
Haileybury College in Herford which takes ages 13+ isn't marked as secondary. Is that true of all such schools?

Come to that, neither is the Cantonian High school in Cardiff marked as SY.

**Comment by iantasker** [comment]
@davehodg 

Are you referring to [Haileybury and Imperial Service College](https://get-information-schools.service.gov.uk/Establishments/Establishment/Details/117607)?

If so its, listed as a 11-18 independent school.

**Comment by davehodg** [comment]
In which field? Phase is NULL.

**Comment by iantasker** [comment]
@davehodg 

I have checked the record on gov.uk via the link above and they list Phase as "Not applicable", as the SIF PhaseType list doesn't have an equivalent we omit the element.

**Comment by davehodg** [comment]
Hm. That ends up mixing primary and secondary schools. Oh well. Better than nothing.

**Comment by davehodg** [comment]
Hm. Means I have to scrape all the data again. Any chance of just making it available as a file?

**Comment by drjonnicholson** [comment]
The original source data is available from Edubase as a CSV (as of today this raw data is http://ea-edubase-api-prod.azurewebsites.net/edubase/edubasealldata20180418.csv).

The most recent data dump from our system is available https://opendata.education/export/SchoolInfos.2018-04-17.xml (big download > 200mb)

We made the initial design decision to represent the DfE data as closely as possible in the first case (we may well revisit this decision later). As the DfE appears to distinguish between an establishment providing "secondary phase" education as opposed to what we'd colloquially refer to as a "secondary school", we decided best not to mix the two notions. However, making a categorisation of SchoolInfo objects that is appropriate to your use case, based on the `NCYearGroupList` data, should be pretty straight forward.

There are plans in the UK A4L (https://www.a4l.org/) community to release a new version of the UK SIF data model. Version 2.0 was released some time ago before free schools were a thing, similar with academies (hence our 2.x endpoints). The A4L (and we as members) welcome any/all feedback.
